packages/atomic/cypress/integration/result-list/result-list.cypress.ts

```diff
@@ -1,158 +1,208 @@
 import * as CommonAssertions from '../common-assertions';
 
-describe('Result List Component', () => {
-  it('should load', () => {
-    new TestFixture().with(addResultList()).init();
-    ResultListSelectors.result().should('have.length.above', 0);
-  });
-
-  describe('when no first search has yet been executed', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .withoutFirstAutomaticSearch()
-        .with(addResultList())
-        .init();
-    });
-
-    it('should render placeholder components', () => {
-      ResultListSelectors.placeholder().should('be.visible');
-    });
-  });
-
-  describe('when an initial search is executed', () => {
-    it('should render the correct number of results', () => {
-      new TestFixture().with(addResultList()).init();
-      ResultListSelectors.result().should('have.length', 10);
-    });
-  });
-
-  describe('when multiple searches are executed', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .with(addResultList())
-        .withElement(generateComponentHTML(pagerComponent))
-        .init();
-    });
-
-    it('should update the results', () => {
-      let firstResultHtml: string;
-
-      ResultListSelectors.firstResult().then((element) => {
-        firstResultHtml = element[0].innerHTML;
-      });
-
-      PagerSelectors.buttonNext().click();
-      cy.wait(500);
-
-      ResultListSelectors.firstResult().should((element) => {
-        const secondResultHtml = element[0].innerHTML;
-        expect(secondResultHtml).not.to.equal(firstResultHtml);
-      });
-    });
-  });
-
-  describe('with elements to measure line height', () => {
-    const lineHeightSelector = '#line-height-el';
-
-    function generateLineHeightElement() {
-      return generateComponentHTML('div', {
-        id: lineHeightSelector.slice(1),
-        style:
-          'background-color: red; width: var(--line-height, 0); height: var(--line-height, 0);',
-      });
-    }
-
-    before(() => {
-      new TestFixture()
-        .with(
-          addResultList(
-            buildTemplateWithSections({
-              title: generateLineHeightElement(),
-              excerpt: generateLineHeightElement(),
-              bottomMetadata: generateLineHeightElement(),
-            })
-          )
-        )
-        .init();
-      ResultListSelectors.shadow().find('.list-wrapper:not(.placeholder)');
-    });
-
-    withAnySectionnableResultList((display, imageSize, density) => {
-      it('should expose --line-height in the title section', () => {
-        ResultListSelectors.sections
-          .title()
-          .find(lineHeightSelector)
-          .should('be.visible');
+const foldedResultListConfig = [
+  FoldedResultListSelectors,
+  foldedResultListComponent,
+  addFoldedResultList,
+  'Folded result List Component',
+  true,
+];
+const resultListConfig = [
+  ResultListSelectors,
+  resultListComponent,
+  addResultList,
+  'Result List Component',
+  false,
+];
+
+const configs: Array<
```

Comment:
Kinda nitpicky, but instead of this being an array for configuration, it could have been an object.

Purely so there's key (with names) to describe what each property represent. So the next time we need to modify this, it's a bit more straightforward versus having to map on line 43 what each value represent.

```
const foldedResultListConfig = {
  componentSelectors: FoldedResultListSelectors,
  componentTag: foldedResultListComponent,
  addResultFn: addFoldedResultList,
  title: 'Folded result List Component',
  isFolded: true,
};
```